azure-devops/projects/pagopa-projects/pagopa-api-config.tf

```diff
@@ -1,281 +1,102 @@
-    name           = "SONARCLOUD_SERVICE_CONN"
-    value          = var.pagopa-api-config.pipeline.sonarcloud.service_connection
-    allow_override = false
-  }
-
-  variable {
-    name           = "SONARCLOUD_ORG"
-    value          = var.pagopa-api-config.pipeline.sonarcloud.org
-    allow_override = false
-  }
-
-  variable {
-    name           = "SONARCLOUD_PROJECT_KEY"
-    value          = var.pagopa-api-config.pipeline.sonarcloud.project_key
-    allow_override = false
-  }
-
-  variable {
-    name           = "SONARCLOUD_PROJECT_NAME"
-    value          = var.pagopa-api-config.pipeline.sonarcloud.project_name
-    allow_override = false
-  }
-}
-
-# code review serviceendpoint authorization
-resource "azuredevops_resource_authorization" "pagopa-api-config-code-review-github-ro-auth" {
-  depends_on = [
-    azuredevops_serviceendpoint_github.io-azure-devops-github-ro,
-    azuredevops_build_definition.pagopa-api-config-code-review, azuredevops_project.project
-  ]
-
-  project_id    = azuredevops_project.project.id
-  resource_id   = azuredevops_serviceendpoint_github.io-azure-devops-github-ro.id
-  definition_id = azuredevops_build_definition.pagopa-api-config-code-review.id
-  authorized    = true
-  type          = "endpoint"
-}
-
-resource "azuredevops_resource_authorization" "pagopa-api-config-code-review-github-pr-auth" {
-  depends_on = [
-    azuredevops_serviceendpoint_github.io-azure-devops-github-pr,
-    azuredevops_build_definition.pagopa-api-config-code-review, azuredevops_project.project
-  ]
-
-  project_id    = azuredevops_project.project.id
-  resource_id   = azuredevops_serviceendpoint_github.io-azure-devops-github-pr.id
-  definition_id = azuredevops_build_definition.pagopa-api-config-code-review.id
-  authorized    = true
-  type          = "endpoint"
-}
-
-# deploy
-resource "azuredevops_build_definition" "pagopa-api-config-deploy" {
-  depends_on = [azuredevops_serviceendpoint_github.io-azure-devops-github-rw, azuredevops_project.project]
-
-  project_id = azuredevops_project.project.id
-  name       = "${var.pagopa-api-config.repository.name}.deploy"
-  path       = "\\${var.pagopa-api-config.repository.name}"
-
-  repository {
-    repo_type             = "GitHub"
-    repo_id               = "${var.pagopa-api-config.repository.organization}/${var.pagopa-api-config.repository.name}"
-    branch_name           = var.pagopa-api-config.repository.branch_name
-    yml_path              = "${var.pagopa-api-config.repository.pipelines_path}/deploy-pipelines.yml"
-    service_connection_id = azuredevops_serviceendpoint_github.io-azure-devops-github-rw.id
-  }
-
-  variable {
-    name           = "GITHUB_CONNECTION"
-    value          = azuredevops_serviceendpoint_github.io-azure-devops-github-rw.service_endpoint_name
-    allow_override = false
+locals {
+  # global vars
+  pagopa-api-config-variables = {
+    cache_version_id = "v1"
+    default_branch   = var.pagopa-mock-ec.repository.branch_name
```

Comment:
this variable is wrong. 
The right value is: `default_branch   = var.pagopa-api-config.repository.branch_name`

Reply:
sure, done